src/slice_transform.rs

```diff
@@ -128,10 +118,6 @@ pub unsafe extern "C" fn in_domain_callback(
 ) -> u8 {
     let cb = &mut *(raw_cb as *mut TransformCallback);
     let key = slice::from_raw_parts(raw_key as *const u8, key_len as usize);
-
-    if (cb.in_domain_fn.unwrap())(key) {
-        1
-    } else {
-        0
-    }
+    let in_domain = cb.in_domain_fn.unwrap();
```

Comment:
Same here

Reply:
This is called by RocksDB when we've provided an `in_domain` callback. `in_domain_fn` being `None` here should result in a loud error. If you think it makes it better we could call `unreachable!()`.